<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Selected publications</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Odiscé</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="team.html">Team</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Selected publications</h1>

</div>


<ul>
<li><p>Roquencourt C., Grassin-Delyle S. and Thevenot, E. A. (2022). ptairMS: real-time processing and analysis of PTR-TOF-MS data for biomarker discovery in exhaled breath. <em>Bioinformatics</em> (<a href="https://doi.org/10.1093/bioinformatics/btac031" class="uri">https://doi.org/10.1093/bioinformatics/btac031</a>)</p></li>
<li><p>Imbert A., Rompais M., Selloum M., Castelli F., Mouton-Barbosa E., Brandolini-Bunlon M., Chu-Van E., Joly C., Hirschler A., Roger P., Burger T., Leblanc S., Sorg T., Ouzia S., Vandenbrouck Y., Medigue C., Junot C., Ferro M., Pujos-Guillot E., de Peredo A. G., Fenaille F., Carapito C., Herault Y. and Thevenot, E. A. (2021). ProMetIS, deep phenotyping of mouse models by combined proteomics and metabolomics analysis. <em>Scientific Data</em>, <strong>8</strong> (<a href="https://doi.org/10.1038/s41597-021-01095-3" class="uri">https://doi.org/10.1038/s41597-021-01095-3</a>)</p></li>
<li><p>Grassin-Delyle S., Roquencourt C., Moine P., Saffroy G., Carn S., Heming N., Fleuriet J., Salvator H., Naline E., Couderc L.-J., Devillier P., Thevenot E. A. and Annane D. (2021). Metabolomics of exhaled breath in critically ill COVID-19 patients: A pilot study. <em>EBioMedicine</em>, <strong>63</strong> (<a href="https://doi.org/10.1016/j.ebiom.2020.103154" class="uri">https://doi.org/10.1016/j.ebiom.2020.103154</a>)</p></li>
<li><p>Stanstrup J., Broeckling C.D., Helmus R., Hoffmann N., Mathe E., Naake T., Nicolotti L., Peters K., Rainer J., Salek R., Schulze T., Schymanski E.L., Stravs M.A., Thevenot E.A., Treutler H., Weber R., Willighagen E., Witting M., Neumann S. The metaRbolomics toolbox in Bioconductor and beyond. <em>Metabolites</em>, <strong>9</strong> (<a href="https://doi.org/10.3390/metabo9100200" class="uri">https://doi.org/10.3390/metabo9100200</a>)</p></li>
<li><p>Emami Khoonsari P., Moreno P., Bergmann S., Burman J., Capuccini M., Carone M., Cascante M., de Atauri P., Foguet C., Gonzalez-Beltran A., Hankemeier T., Haug K., He S., Herman S., Johnson D., Kale N., Larsson A., Neumann S., Peters K., Pireddu L., Rocca-Serra P., Roger P., Rueedi R., Ruttkies C., Sadawi N., Salek R.M., Sansone S.A., Schober D., Selivanov V., Thevenot E.A., van Vliet M., Zanetti G., Steinbeck C., Kultima K. and Spjuth O. (2019). Interoperable and scalable data analysis with microservices: applications in metabolomics. <em>Bioinformatics</em>, <strong>35</strong>:3752-3760 (<a href="https://doi.org/10.1093/bioinformatics/btz160" class="uri">https://doi.org/10.1093/bioinformatics/btz160</a>)</p></li>
<li><p>Peters K., Bradbury J., Bergmann S., Capuccini M., Cascante M., de Atauri P., Ebbels T.M.D., Foguet C., Glen R., Gonzalez-Beltran A., Gunther U.L., Handakas E., Hankemeier T., Haug K., Herman S., Holub P., Izzo M., Jacob D., Johnson D., Jourdan F., Kale N., Karaman I., Khalili B., Emami Khonsari P., Kultima K., Lampa S., Larsson A., Ludwig C., Moreno P., Neumann S., Novella J.A., O’Donovan C., Pearce J.T.M., Peluso A., Piras M.E., Pireddu L., Reed M.A.C., Rocca-Serra P., Roger P., Rosato A., Rueedi R., Ruttkies C., Sadawi N., Salek R.M., Sansone S.A., Selivanov V., Spjuth O., Schober D., Thevenot E.A., Tomasoni M., van Rijswijk M., van Vliet M., Viant M.R., Weber R.J.M., Zanetti G. and Steinbeck C. (2019). PhenoMeNal: processing and analysis of metabolomics data in the cloud. <em>Gigascience</em>, <strong>8</strong> (<a href="https://doi.org/10.1093/gigascience/giy149" class="uri">https://doi.org/10.1093/gigascience/giy149</a>)</p></li>
<li><p>van Rijswijk M., Beirnaert C., Caron C., Cascante M., Dominguez V., Dunn W., Ebbels T., Giacomoni F., Gonzalez-Beltran A., Hankemeier T., Haug K., Izquierdo-Garcia J., Jimenez R., Jourdan F., Kale N., Klapa M., Kohlbacher O., Koort K., Kultima K., Le Corguille G., Moschonas N., Neumann S., O’Donovan C., Reczko M., Rocca-Serra P., Rosato A., Salek R., Sansone S., Satagopam V., Schober D., Shimmo R., Spicer R., Spjuth O., Thevenot E., Viant M., Weber R., Willighagen E., Zanetti G. and Steinbeck C. (2017). The future of metabolomics in ELIXIR. <em>F1000Research</em>, <a href="doi:%5B10.12688/f1000research.12342.1" class="uri">doi:[10.12688/f1000research.12342.1</a>](<a href="https://doi.org/10.12688/f1000research.12342.1" class="uri">https://doi.org/10.12688/f1000research.12342.1</a>)</p></li>
<li><p>Delabriere A., Hohenester U.M., Colsch B., Junot C., Fenaille F. and Thevenot E.A. (2017). <em>proFIA</em>: a data preprocessing workflow for flow injection analysis coupled to high-resolution mass spectrometry. <em>Bioinformatics</em>, <strong>33</strong>:3767-3775 (<a href="https://doi.org/10.1093/bioinformatics/btx458" class="uri">https://doi.org/10.1093/bioinformatics/btx458</a>)</p></li>
<li><p>Guitton Y., Tremblay-Franco M., Le Corguille G., Martin J.-F., Petera M., Roger-Mele P., Delabriere A., Goulitquer S., Monsoor M., Duperier C., Canlet C., Servien R., Tardivel P., Caron C., Giacomoni F. and Thevenot, E.A. (2017). Create, run, share, publish, and reference your LC-MS, FIA-MS, GC-MS, and NMR data analysis workflows with the Workflow4Metabolomics 3.0 Galaxy online infrastructure for metabolomics. <em>International Journal of Biochemistry and Cell Biology</em>, <strong>93</strong>:89-101 (<a href="https://doi.org/10.1016/j.biocel.2017.07.002" class="uri">https://doi.org/10.1016/j.biocel.2017.07.002</a>)</p></li>
<li><p>Rinaudo P., Boudah S., Junot C. and Thevenot E.A. (2016). <em>biosigner</em>: a new method for the discovery of significant molecular signatures from omics data. <em>Frontiers in Molecular Biosciences</em>, <strong>3</strong> (<a href="https://doi.org/10.3389/fmolb.2016.00026" class="uri">https://doi.org/10.3389/fmolb.2016.00026</a>)</p></li>
<li><p>Thevenot E.A., Roux A., Xu Y., Ezan E. and Junot C. (2015). Analysis of the human adult urinary metabolome variations with age, body mass index and gender by implementing a comprehensive workflow for univariate and OPLS statistical analyses. <em>Journal of Proteome Research</em>, <strong>14</strong>:3322-3335 (<a href="https://doi.org/10.1021/acs.jproteome.5b00354" class="uri">https://doi.org/10.1021/acs.jproteome.5b00354</a>)</p></li>
<li><p>Giacomoni F., Le Corguille G., Monsoor M., Landi M., Pericard P., Petera M., Duperier C., Tremblay-Franco M., Martin J.-F., Jacob D., Goulitquer S., Thevenot E.A. and Caron C. (2015). Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics. <em>Bioinformatics</em>, <strong>31</strong>:1493-1495 (<a href="https://doi.org/10.1093/bioinformatics/btu813" class="uri">https://doi.org/10.1093/bioinformatics/btu813</a>)</p></li>
<li><p>Lenuzza N., Duval X., Nicolas G., Thevenot E., Job S., Videau O., Narjoz C., Loriot M.-A., Beaune P., Becquemont L., Mentre F., Funck-Brentano C., Alavoine L., Arnaud P., Delaforge M. and Benech H. (2014). Safety and pharmacokinetics of the CIME combination of drugs and their metabolites after a single oral dosing in healthy volunteers. <em>European Journal of Drug Metabolism and Pharmacokinetics</em>, 1-14 (<a href="https://doi.org/10.1007/s13318-014-0239-0" class="uri">https://doi.org/10.1007/s13318-014-0239-0</a>)</p></li>
</ul>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
