<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Odiscé</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="team.html">Team</a>
</li>
<li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="projects.html">Projects</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<p>Our research has been focusing for more than 15 years on
<strong>omics data sciences</strong> for <strong>systems biology and
precision medicine</strong>, with a particular emphasis on metabolomics
data sciences.</p>
<p><strong>Data sciences</strong> provide powerful approaches and
algorithms (signal processing, data mining, machine learning, artificial
intelligence) for the <strong>processing and analysis of
high-dimensional data</strong>, such as omics datasets.
<strong>Metabolomics</strong> (untargeted analysis of small molecules
involved in biochemical reactions) is of major interest for phenotype
characterization and biomarker discovery. <strong>High-resolution mass
spectrometry (HRMS)</strong> is a technology of choice for metabolomics
(and also for proteomics), due to its sensitivity and resolution.</p>
<p>We have implemented our new methods in a <strong>comprehensive
digital ecosystem</strong> for <strong>metabolomics data
sciences</strong>, with applications to <strong>precision
medicine</strong> (including neurosciences, liver diseases and food
allergy).</p>
<div class="figure">
<img src="images/odisce_software.png" alt="" />
<p class="caption">Software suite for metabolomics data sciences</p>
</div>
<div id="data-processing" class="section level2">
<h2>Data processing</h2>
<p><strong>Direct injection methods</strong> (such as Flow Injection
Analysis or Proton Transfer Reaction) are of particular interest for
high-throughput phenotyping. We therefore developed an innovative
<strong>preprocessing workflow</strong> which takes as input the
individual raw files and generates the samples by variables table of
intensities (peak table). The steps include (i) <strong>peak detection
and quantification</strong> within each file, (ii) <strong>peak
alignment</strong> across samples to generate the peak table, and (iii)
<strong>missing value imputation</strong>. In particular, new methods
were required to optimize step (i), including noise estimation, modeling
of the injection peak, and precise determination of each analyte peak
borders. Application to several real data sets resulted in robust and
accurate detection and quantification. <a
href="http://bioconductor.org/packages/proFIA"><strong>proFIA</strong></a>
is available as an R/Bioconductor package (<a
href="http://dx.doi.org/10.1093/bioinformatics/btx458"><strong>Delabrière
et al, 2017</strong></a>).</p>
<div class="figure">
<img src="images/proFIA.png" alt="" />
<p class="caption"><a
href="http://bioconductor.org/packages/proFIA"><strong>proFIA</strong></a>
workflow for the processing of high-throughput and high-resolution
FIA-HRMS data (<a
href="http://dx.doi.org/10.1093/bioinformatics/btx458"><strong>Delabrière
et al, 2017</strong></a>)</p>
</div>
<p>In <strong>volatolomics</strong> (analysis of volatile organic
compounds), PTR-TOF-MS offers unique opportunities for <strong>real-time
analysis of exhaled air at the patient’s bedside</strong> (<a
href="http://dx.doi.org/10.1016/j.ebiom.2020.103154"><strong>Grassin
Delyle et al., 2021</strong></a><strong>)</strong>. We therefore
developed a <strong>comprehensive suite of algorithms for the
pre-processing of acquisitions in large cohorts</strong>, which includes
an <strong>innovative two-dimensional peak deconvolution model</strong>
based on penalized splines signal regression for accurate estimation of
the temporal profile, implemented in the <a
href="http://bioconductor.org/packages/ptairMS"><strong>ptairMS</strong></a>
software (<a
href="https://doi.org/10.1093/bioinformatics/btac031"><strong>Roquencourt
et al, 2022</strong></a>).</p>
<div class="figure">
<img src="images/ptairMS.png" alt="" />
<p class="caption">Main steps of the processing of PTR-TOF-MS data with
the <a
href="http://bioconductor.org/packages/ptairMS"><strong>ptairMS</strong></a>
software (<a
href="https://doi.org/10.1093/bioinformatics/btac031"><strong>Roquencourt
et al, 2022</strong></a>)</p>
</div>
</div>
<div id="data-modeling" class="section level2">
<h2>Data modeling</h2>
<p>We implemented the <strong>Orthogonal Partial Least-Squares
(OPLS)</strong> approach for regression and classification from <a
href="http://dx.doi.org/10.1002/cem.695">Trygg and Wold (2002)</a> as an
R package named <a
href="http://bioconductor.org/packages/ropls"><strong>ropls</strong></a>
(<a
href="https://doi.org/10.1021/acs.jproteome.5b00354"><strong>Thévenot et
al, 2015</strong></a>). OPLS algorithm is a variation of PLS and allows
to model separately the orthogonal variation (i.e. non-correlated to the
response) and the predictive variation (i.e. correlated to the
response), and thus facilitates model interpretation.</p>
<div class="figure">
<img src="images/ropls.jpg" alt="" />
<p class="caption">PCA, PLS(-DA) and OPLS(-DA) modeling with <a
href="http://bioconductor.org/packages/ropls"><strong>ropls</strong></a>
(<a
href="http://dx.doi.org/10.1021/acs.jproteome.5b00354"><strong>Thévenot
et al, 2015</strong></a>)</p>
</div>
<p>We developed a new methodology for <strong>feature
selection</strong>, of the wrapper type, which assesses the significance
of the features for the model performance (<a
href="http://bioconductor.org/packages/biosigner"><strong>biosigner</strong></a>
R package). The wrapping of three classifiers (PLS-DA, Random Forest and
Support Vector Machine) with this methodology resulted in
<strong>stable</strong> signatures of <strong>restricted size</strong>,
when applied to real metabolomics and transcriptomics datasets (<a
href="https://doi.org/10.3389/fmolb.2016.00026"><strong>Rinaudo et al,
2016</strong></a>).</p>
<div class="figure">
<img src="images/biosigner.jpg" alt="" />
<p class="caption">Selection of significant omics features with <a
href="http://bioconductor.org/packages/biosigner"><strong>biosigner</strong></a>
for PLS-DA, Random Forest or SVM prediction (<a
href="https://doi.org/10.3389/fmolb.2016.00026"><strong>Rinaudo et al,
2016</strong></a>)</p>
</div>
<p><strong>Integration of complementary omics data</strong> is an
opportunity to build <strong>more robust predictive models</strong> and
<strong>facilitate the biologicial interpretation</strong>. To
demonstrate the feasibility and interest of <strong>combining proteomics
and metabolomics</strong> in routine, we have developed, within a
consortium of research infrastructures in <strong>phenogenomics (<a
href="http://www.phenomin.fr/en-us/">PHENOMIN</a>)</strong>,
<strong>proteomics (<a
href="http://www.profiproteomics.fr/">ProFI</a>)</strong>,
<strong>metabolomics (<a
href="https://www.metabohub.fr/home.html">MetaboHUB</a>)</strong> and
<strong>informatics (<a
href="http://www.france-bioinformatique.fr/en">IFB</a>)</strong>, a
<strong>data post-processing and integration pipeline</strong> that we
have applied to the study of murine models. All the data and codes are
available in the <a
href="https://github.com/IFB-ElixirFr/ProMetIS"><strong>ProMetIS</strong></a>
package (<a
href="https://doi.org/10.1038/s41597-021-01095-3"><strong>Imbert et al.,
2021</strong></a>).</p>
<div class="figure">
<img src="images/ProMetIS.png" alt="" />
<p class="caption"><a
href="https://github.com/IFB-ElixirFr/ProMetIS"><strong>ProMetIS</strong></a>,
Deep phenotyping of mouse models by combined proteomics and metabolomics
analysis (<a
href="https://doi.org/10.1038/s41597-021-01095-3"><strong>Imbert et al.,
2021</strong></a>)</p>
</div>
</div>
<div id="computing-with-data" class="section level2">
<h2>Computing with data</h2>
<p><strong>Reference datasets</strong> are available in the <a
href="https://www.ebi.ac.uk/metabolights/"><strong>MetaboLights</strong></a>
repository as well as in R packages (<a
href="https://www.bioconductor.org/packages/plasFIA/"><strong>plasFIA</strong></a>,
<a
href="https://www.bioconductor.org/packages/ptairData/"><strong>ptairData</strong></a>,
<a
href="http://bioconductor.org/packages/ropls"><strong>ropls</strong></a>,
<a
href="http://bioconductor.org/packages/biosigner"><strong>biosigner</strong></a>,
<a
href="https://github.com/IFB-ElixirFr/ProMetIS"><strong>ProMetIS</strong></a>).</p>
<p>All algorithms are implemented as a <strong>comprehensive suite of 6
R packages</strong> on the <a
href="http://bioconductor.org"><strong>Bioconductor</strong></a>
repository.</p>
<p>Many of them are also available as <strong>Galaxy modules</strong> on
the <a
href="http://workflow4metabolomics.org"><strong>Workflow4Metabolomics</strong></a>
online platform,jointly developed and maintained by the <a
href="http://www.france-bioinformatique.fr/en"><strong>French Institute
of Bioinformatics</strong></a> and <a
href="http://www.metabohub.fr/home.html"><strong>MetaboHUB</strong></a>
(<a
href="https://doi.org/10.1093/bioinformatics/btu813"><strong>Giacomoni
et al, 2015</strong></a>; <a
href="https://doi.org/10.1016/j.biocel.2017.07.002"><strong>Guitton et
al, 2017</strong></a>).</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
